Stripe/STPBankAccount.m

```diff
@@ -54,7 +54,64 @@ - (BOOL)isEqualToBankAccount:(STPBankAccount *)bankAccount {
     return [self.bankAccountId isEqualToString:bankAccount.bankAccountId];
 }
 
-#pragma mark STPAPIResponseDecodable
+#pragma mark - Description
+
+- (NSString *)description {
+    NSString *statusDescription;
+
+    switch (self.status) {
+        case STPBankAccountStatusNew:
+            statusDescription = @"new";
+            break;
+        case STPBankAccountStatusValidated:
+            statusDescription = @"validated";
+            break;
+        case STPBankAccountStatusVerified:
+            statusDescription = @"verified";
+            break;
+        case STPBankAccountStatusErrored:
+            statusDescription = @"errored";
+            break;
+    }
+
+    NSString *accountHolderTypeDescription;
+
+    switch (self.accountHolderType) {
+        case STPBankAccountHolderTypeIndividual:
+            accountHolderTypeDescription = @"individual";
+            break;
+        case STPBankAccountHolderTypeCompany:
+            accountHolderTypeDescription = @"company";
+            break;
+    }
+
+    NSArray *props = @[
+                       // Object
+                       [NSString stringWithFormat:@"%@: %p", NSStringFromClass([self class]), self],
+
+                       // Identifier
+                       [NSString stringWithFormat:@"bankAccountId = %@", self.bankAccountId],
```

Comment:
Do you mean something like this?

```
[NSString stringWithFormat:@"%@ = %@", NSStringFromSelector(bankAccountId), self.bankAccountId],
```

I think I like the clarity with what I have here because it breaks down when we have something generated like:

```
[NSString stringWithFormat:@"status = %@", statusDescription]
```